pycsw/ogc/csw/csw3.py

```diff
@@ -514,9 +519,8 @@ def getdomain(self):
                     operation, parameter = pname.split('.')
                 except:
                     return node
-                if (operation in self.parent.context.model['operations'].keys() and
-                    parameter in
-                    self.parent.context.model['operations'][operation]['parameters'].keys()):
+                if (operation in list(self.parent.context.model['operations'].keys()) and
+                    parameter in list(self.parent.context.model['operations'][operation]['parameters'].keys())):
```

Comment:
Can drop the `list` and `.keys()`, and this second line is not indented enough for PEP8.